client/devdocs/docs-example/style.scss

```diff
@@ -0,0 +1,65 @@
+.docs-example__header {
+	display: flex;
+	align-items: center;
+}
+
+.docs-example__title {
+	flex: 1;
+}
+
+.docs-example__title,
+.docs-example__toggle {
+	margin: 0 !important;
+}
+
+.docs-example__header + .docs-example__main {
```

Comment:
I've no idea. Let me take a look but keep in mind in this case I've just moved the `docs-example` component from `client/components` to `devdocs/`.